### PR TITLE
Navigator 2.3.9 bump

### PR DESCRIPTION
**Pull request by ChrisKraljevicD2L**

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -25397,4 +25400,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```
Comment (sarahboyd) [inline package-lock.json]: I don't think this pulled in the new navigator version

Comment (ChrisKraljevicD2L) [reply package-lock.json]: 🤦‍♂ yup. Imma fix that

Comment (ChrisKraljevicD2L) [reply package-lock.json]: I also screwed something up in navigator that I now noticed running the demo. Gonna make another pr over there and update this one

Comment (ChrisKraljevicD2L) [reply package-lock.json]: [Tadaaa](https://github.com/BrightspaceHypermediaComponents/d2l-sequence-navigator/pull/183)

File: package-lock.json [hunk 2]
```diff
@@ -2820,7 +2870,7 @@
       }
     },
     "d2l-sequences": {
-      "version": "github:BrightspaceHypermediaComponents/sequences#78338c62f2b4f4f1cec75a14da902287d54d7e1a",
+      "version": "github:BrightspaceHypermediaComponents/sequences#4cce47a8860abf1f7c8a3781e7f9000caf333570",
```
Comment (sarahboyd) [inline package-lock.json]: wanna do one more update to bring in Phil's sequence changes? 😛  https://github.com/BrightspaceHypermediaComponents/sequences/releases/tag/v1.3.16

**Pull request merged**